verl/models/mcore/model_initializer.py

```diff
@@ -156,6 +157,29 @@ def initialize(self, **kwargs):
         return model
 
 
+class DeepseekV3Model(BaseModelInitializer):
+    """Initializer for DeepseekV3 models."""
+
+    def get_transformer_layer_spec(self):
+        assert self.tfconfig.normalization == "RMSNorm", "only RMSNorm is supported for now"
```

Comment:
can we make ``RMSNorm is supported for now`` clear like current TransformerEngine or other lib can not support?